README.md

```diff
@@ -1,57 +1,10 @@
 # Northcoders House of Games API
 
-## Background
+## Add Setup Files:
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+- Add .env files (both test and development) for relevant database
```

Comment:
Might be worth elaborating that the .env files will be used to set the PGDATABASE for the two different environments